src/engine/Drawing/SpriteSheet.ts

```diff
@@ -98,6 +101,75 @@ export class SpriteSheetImpl {
     }
   }
 
+  /**
+   * Gets the raw spacing dimensions for the sprites in the sheet, which can be a fixed number or custom dimensions.
+   */
+  public get spacing() {
+    return this._spacing;
+  }
+
+  /**
+   * Get a copy of the backing sprite array
+   */
+  public get sprites() {
+    return [...this._sprites];
```

Comment:
@eonarheim should I do this instead?

```suggestion
    return this._sprites.map(sprite => sprite.clone());
```